app/app/src/main/res/layout/content_joystick.xml

```diff
@@ -1,11 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/joystickContent"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
-    android:visibility="gone" >
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <!--com.erz.joystick.GameView
+        android:id="@+id/game"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" -->
+
+    <com.example.myapplication.JoyStick
+        android:id="@+id/joy1"
+        android:layout_width="200dp"
+        android:layout_height="200dp"
+        android:layout_gravity="bottom"/>
+
+    <com.example.myapplication.JoyStick
```

Comment:
Why are there two joysticks?

Reply:
The left joystick is responsible for forward, back, left and right direction. And the right is planned for more direction. If forward,back, left and right is enough, then I can remove the right one.

Reply:
"As a user, I want the smart-car steering to be achievable with only one hand, so that I can switch my attention between the car and the phone screen."
The goal of the user story for the related issue is that the user should be able to manually control the car with only one and. If you include two different joysticks the user has to use both their hands.